src/registry_schemas/example_data/schema_data.py

```diff
@@ -2568,6 +2568,43 @@
     }
 }
 
+CORRECTION_CP_SPECIAL_RESOLUTION = {
```

Comment:
Please add the `type` field.  Maybe something like `'type': 'CLIENT'`.

`rulesUploadedOn`, `provisionsRemoved`, `resolutionConfirmed`, `subType` can be removed from the example data.  I'm assuming most of these are just used by the FE or not relevant to SR corrections.